extensions/wikia/FounderEmails/FounderEmails.class.php

```diff
@@ -215,23 +215,20 @@ public function getPageViews ( $cityID ) {
 	}
 
 	public function getDailyEdits ( $cityID, /*Y-m-d*/ $day = null ) {
-		global $wgStatsDB, $wgStatsDBEnabled;
+		global $wgDWStatsDB;
 
-		$edits = 0;
-		if ( !empty( $wgStatsDBEnabled ) ) {
-			$today = ( empty( $day ) ) ? date( 'Y-m-d', strtotime( '-1 day' ) ) : $day;
+		$today = ( empty( $day ) ) ? date( 'Y-m-d', strtotime( '-1 day' ) ) : $day;
 
-			$db = wfGetDB( DB_SLAVE, array(), $wgStatsDB );
+		$db = wfGetDB( DB_SLAVE, array(), $wgDWStatsDB );
 
-			$oRow = $db->selectRow(
-				array( 'events' ),
-				array( 'count(0) as cnt' ),
-				array(  " rev_timestamp between '$today 00:00:00' and '$today 23:59:59' ", 'wiki_id' => $cityID ),
-				__METHOD__
-			);
+		$oRow = $db->selectRow(
```

Comment:
We can use `$db->selectField` helper here

Reply:
:+1: